elytron-oidc-client/subsystem/src/test/java/org/wildfly/extension/elytron/oidc/OidcTestCase.java

```diff
@@ -50,11 +51,17 @@ public OidcTestCase() {
     @Before
     public void prepare() throws Throwable {
         if (services != null) return;
+        if (servicesExpression != null) return;
         String subsystemXml = "oidc.xml";
+        String subsystemExpressionXml = "oidc-expressions.xml";
         services = super.createKernelServicesBuilder(new DefaultInitializer()).setSubsystemXmlResource(subsystemXml).build();
         if (! services.isSuccessfulBoot()) {
             Assert.fail(services.getBootError().toString());
         }
+        servicesExpression = super.createKernelServicesBuilder(new DefaultInitializer()).setSubsystemXmlResource(subsystemExpressionXml).build();
+        if (! servicesExpression.isSuccessfulBoot()) {
+            Assert.fail(servicesExpression.getBootError().toString());
+        }
```

Comment:
@Ashpan Thanks very much for working on this! Do you think it would be possible to move this to the `Subsystem_1_0_ParsingTestCase`? The `OidcTestCase` was intended more for verifying that subsystem configuration can be successfully converted into JSON config that's associated with the deployment.

Reply:
Moved to it's own test in `ExpressionsTestCase`